added entry color changes

sanatankc opened the pull request:
Pull request is not complete at the moment. Because I am not able to test them locally.

TODO
- [ ] to test that these changes are actually working and entry is black instead of white.
- [x] Fix white border bug.

quozl commented on gtk3/theme/gtk-widgets.css.em:
```diff
@@ -196,6 +196,10 @@ spinbutton button:last-child,
 }
 
+spinbutton entry,
+.spinutton.entry {
```
You wrote here `spinutton`, presumably a class of elderly sheep that has a habit of hiding in spinifex. :grin:

Ba-aa!

sanatankc replied:
Oh sorry 😄

sanatankc commented:
Thanks to @quozl @i5o @walterbender, I finally figured out that changes are actually need to be done in `3.20`. I have removed white border and additionally also fixed buttons height and margin.

![screen shot 2017-12-20 at 5 39 39 pm](https://user-images.githubusercontent.com/20373641/34206680-8f66180c-e5ad-11e7-8af2-bf8e937e8e40.png)

Weirdly, entry's color is not fixing. I tried
```
spinbutton entry {
    color: #000000;
}
```
Though, setting `background-color`, works.
```
spinbutton entry {
    background-color: #000000;
}
```

sanatankc commented:
On `entry` no css related to Text is working. I thinks is in different node inside entry node. But can't say what. 

### Alternative
It is bit radical. But if we couldn't able to change text color, why don't we change background of entry.
Like this:
![screen shot 2017-12-20 at 9 41 19 pm](https://user-images.githubusercontent.com/20373641/34235548-6aeac71e-e618-11e7-9aa9-fabc31fa2616.png)
Let me know what you guys think about this, @quozl @walterbender @i5o

quozl commented:
No objection.  Looks cleaner to me.  You might also try changing the - and + foreground colour to differentiate slightly from text foreground colour.

Otherwise, consult with the GTK+ project to figure out how to set the text foreground colour.

sanatankc commented:
Thanks @quozl, I have updated spin button to differentiate between entry color and buttons.

![screen shot 2017-12-21 at 6 48 12 am](https://user-images.githubusercontent.com/20373641/34235998-5bdfa20a-e61b-11e7-9780-32228eddfaac.png)
